aristanetworks/ansible-avd
# #297 splitted the eos and documentation templates virtual-router-mac-…

…address-virtual-source-nat.j2


## Change Summary


## Types of changes


## Related Issue(s)


 #297 
 #322  
## Component(s) name

## Proposed changes


- split the eos template virtual-router-mac-address-virtual-source-nat.j2 into 2 diff eos templates
- split the doc template virtual-router-mac-address-virtual-source-nat.j2 into 2 diff doc templates
- update the required files
   
## How to test



run the role `eos_cli_config_gen` and check the generated eos configuration and the generated documentation (for both virtual-router-mac-address and virtual-source-nat. 

## Checklist:
